bundle/kubernetes/manifests/configmap.yaml

```diff
@@ -554,24 +554,24 @@ data:
     # Choices: PREEMPTIVE NON_PREEMPTIVE
     #failover_mode = NON_PREEMPTIVE
 
-    # Set this to ENABLE to enable NCP enforced pool member limit for all load
+    # Set this to ACTIVATE to enable NCP enforced pool member limit for all load
     # balancer servers in cluster. Set this to CRD_LB_ONLY will only enforce
     # the limit for load balancer servers created using lb CRD. Set this to
-    # DISABLE to turn off all limit checks. This option requires
+    # DEACTIVATE to turn off all limit checks. This option requires
     # relax_scale_validation set to True, l4_lb_auto_scaling set to False, and
-    # works on Policy API only. When not disabled, NCP will enforce a pool
+    # works on Policy API only. When activated, NCP will enforce a pool
     # member limit on LBS to prevent one LBS from using up all resources on
     # edge nodes.
-    # Choices: DISABLE ENABLE CRD_LB_ONLY
-    #ncp_enforced_pool_member_limit = DISABLE
+    # Choices: DEACTIVATE ACTIVATE CRD_LB_ONLY
+    #ncp_enforced_pool_member_limit = DEACTIVATE
```

Comment:
if changing the eum of this value, do we also need to change logic that will use this option?

Reply:
Hi Sean, Thanks for comment. The logic of using this option has been changed as well on NCP side.